src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml

```diff
@@ -4,12 +4,13 @@
        x:CompileBindings="True"
        Selector="NativeMenuBar">
   <Style.Resources>
-    <local:InverseBooleanValueConverter x:Key="AvaloniaThemesDefaultNativeMenuBarInverseBooleanValueConverter" Default="True"/>
+    <local:InverseBooleanValueConverter x:Key="AvaloniaThemesFluentNativeMenuBarInverseBooleanValueConverter" Default="True"/>
+    <local:IBitmapToImageConverter x:Key="AvaloniaThemesFluentNativeMenuBarIBitmapToImageConverterr"/>
```

Comment:
```suggestion
    <local:IBitmapToImageConverter x:Key="AvaloniaThemesFluentNativeMenuBarIBitmapToImageConverter"/>
```
Typo. Also, in other places as well.

Reply:
@maxkatz6 
Upps... You are right :)

Reply:
@Megapit no problem. Can you commit fix of the typo?

Reply:
OK. Done
Did not know that i have to do this. I thought it is allready done by your suggestions.
Next time i know thw flow of making a pull request. I promise 😉

Reply:
Thanks :)